articles/list-transformations-js/index.md

```diff
@@ -0,0 +1,357 @@
+﻿
+## List transformations in JavaScript
+
+In this tutorial, we will be looking at creating new objects and arrays from existing ones. We will look at creating nested data structures, looping and destructuring them. We will also look at transforming data structures by passing them through functions. 
+
+### Prerequisites
+
+To follow through this tutorial, you are required to have at least some knowledge in JavaScript [objects and arrays](https://www.section.io/engineering-education/object-arrays-destructuring/).
```

Comment:
To be grammatically correct consider changing:
"To follow through this tutorial..." to "To follow through with this tutorial..."

Reply:
The same to 
"... some knowledge in JavaScript" to "... some knowledge of JavaScript"